### PR TITLE
flatbuffers@24.3.7

### PR DESCRIPTION
**Pull request by mmorel-35**
Release: https://github.com/google/flatbuffers/releases/tag/v24.3.7

Related to #1604

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (flatbuffers) have been updated in this PR. Please review the changes.

**Comment by mmorel-35** [comment]
Hi @adzenith,

I took example from your PR on flatbuffers to create this

**Comment by adzenith** [comment]
This was the original MR on the Flatbuffers side, just to link everything together.
https://github.com/google/flatbuffers/pull/8227

File: modules/flatbuffers/24.3.7/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+node = use_extension("@rules_nodejs//nodejs:extensions.bzl", "node")
+use_repo(node, "nodejs_linux_amd64")
+
+rules_ts_ext = use_extension("@aspect_rules_ts//ts:extensions.bzl", "ext", dev_dependency = True)
+rules_ts_ext.deps()
+use_repo(rules_ts_ext, "npm_typescript")
+
+non_module_dependencies = use_extension("//:extensions.bzl", "non_module_dependencies")
+use_repo(
+    non_module_dependencies,
```
Comment (meteorcloudy) [inline modules/flatbuffers/24.3.7/MODULE.bazel:71]: Why do we need a Bazel binary as dependency? Is it only a dev dependency for tests?

Comment (meteorcloudy) [reply modules/flatbuffers/24.3.7/MODULE.bazel:71]: Yeah, let's try that.

File: modules/flatbuffers/24.3.7/presubmit.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+buildifier: latest
+bazel: 6.4.0
```
Comment (meteorcloudy) [inline modules/flatbuffers/24.3.7/presubmit.yml:3]: Can you replace with `6.x` to fetch the latest minor release for Bazel 6? (currently 6.5.0)

This module version doesn't work with 7.1?

**Review by meteorcloudy — APPROVED** [review]

**Comment by jjh42** [comment]
Is there a simple example of using this I could look at @mmorel-35 (thanks for this nice work)

**Pull request merged**